python/doc/examples/meta_modeling/kriging_metamodel/plot_kriging_multioutput_firesatellite.py

```diff
@@ -53,7 +54,8 @@
 # We choose to use a constant trend.
```

Comment:
The trend is no longer constant with your changes.

Reply:
exact, need to update it. As I am struggling with other issues, I missed this one, thanks